Update sticker sheet

bradnunnally opened the pull request:
Updated the sticker sheets to include the new header pattern and to correct a "design bug" that was present in the illustrator file where the body copy in the table component was the wrong font size. 

Ready for review @shawnbot and @juliaelman

Commit:
Updated all sticker sheets with Headers

Ready for review by the team.

Commit:
Updated font size on AI file

Updated the table font size to be consistent with our guidance on body
text.

juliasolorzano reviewed (commented):
@bradnunnally nice work! 🎉 Here are a few questions and comments I have, broken out into each file to make it easier to follow:

**ALL**
- Please add the extended header with no mega menu.
- Please add the basic header with the mega menu.
- The extended header with the mega menu should have the top level item highlighted.
- Should we call out the show/hide functionality of the search box in the extended header in the file somehow?
- Please change the font size in the table component from 16px to 17px.

**Sketch, PDF**
- The search icon for the extended header is offset. See the screenshot below for reference: 
![screen shot 2017-02-21 at 1 08 34 pm](https://cloud.githubusercontent.com/assets/955558/23178049/fb3dd8c2-f836-11e6-8a1e-20fc0cc006b7.png)

**Omnigraffle**
- Should we replace the v.8 with v.1.0? Not sure how to get this naming convention in sync with our 1.0 release.

bradnunnally commented:
Sketch file updated. Going to sync the branch then move on to the AI file.

Commit:
Update sketch

Updated the sketch sticker sheet based on feedback from Julia.

Commit:
Updated sticker sheet (Illustrator file)

Updated the illustrator file based on Julia’s feedback.

Commit:
Updated Omnigraffle

Updated the Omnigraffle file based on the feedback from Julia.

Commit:
Updated the exported files

Added PDFs of the omnigraffle and sketch file and an EPS of the
Illustrator file.

bradnunnally commented:
@juliaelman The files have all been updated based on your feedback.

juliasolorzano commented:
Putting some more notes in here to keep track of all the things:

- Please change the font size in the table component from 16px to 17px in all of the files.
- The underline is off a bit in the mega menu selected tab indicator, see screenshot below.
- Remove `Omnigraffle` from PDF filename

![screen_shot_2017-02-22_at_4_19_25_pm](https://cloud.githubusercontent.com/assets/955558/23266132/cfecd810-f9b4-11e6-8983-a9ca2cbb7f96.png)

Commit:
Updated file names and table font

Updated the preview file names to point to what was the source and
corrected the font size on tables to be the correct size.

Commit:
Minor fix on sticker sheets

Fixed where the active underline on the sticker sheets.

Commit:
Update read me

Updated the read me to include instruction on how to use the
Omnigraffle stencil and how to use the preview files.

bradnunnally commented:
Alright! 🎉  The sticker sheets have been updated, file names have been updated, and the readme file has been updated. 

I'm 99% sure everything is good to start merging this PR. 

/cc @juliaelman @shawnbot

juliasolorzano commented on README.md:
```diff
@@ -2,14 +2,16 @@
 This repository hosts editable, open source design files of the [U.S. Web Design Standards](https://playbook.cio.gov/designstandards) for government designers or anyone wishing to make use of the UI components or visual style guide in this site.
 
-For the alpha release, the following file types are available for your use:
+For the 1.0 release, the following file types are available for your use:
```
I think we can remove `For the 1.0 release,` from this sentence.

juliasolorzano commented on README.md:
```diff
@@ -2,14 +2,16 @@
 
 This repository hosts editable, open source design files of the [U.S. Web Design Standards](https://playbook.cio.gov/designstandards) for government designers or anyone wishing to make use of the UI components or visual style guide in this site.
```
Let's update the url in this paragraph to https://standards.usa.gov

juliasolorzano reviewed (changes requested):
I think we are almost there!

- Are we going to have an EPS file too or just the EPS preview? When I pulled down the latest change set it only had the EPS preview file.

Commit:
Update url to Standards website in README.

Commit:
Update EPS file name.

Commit:
Update README copy.

Commit:
Update README copy.

Commit:
Update filename extension for Omnigraffle to make tests work.

juliasolorzano commented:
Done 🚀
